app/Models/Foundation/Summit/Events/Presentations/Presentation.php

```diff
@@ -398,31 +393,99 @@ public function getClassName(): string
      */
     public function getSpeakers()
     {
-        return $this->speakers;
+        return $this->speakers->map(function ($entity) {
```

Comment:
this need to be sorted by order first so u return the collection already ordered

Reply:
done